### PR TITLE
Refactoring gyp files

### PR DESCRIPTION
**Pull request by farfromrefug**
This is allow updating the SDK in only one place

### CODE REVIEW

**Comment by jasongin** [comment]
All the files under `uwp/` are generated by [NodeRT](https://github.com/NodeRT/NodeRT). So, changing the files will make updating them more difficult. We will have to keep updating them as long as we're still using preview SDK version. Technically if you're trying to target a different SDK version, all those files should be regenerated anyway, so you wouldn't need to manually change the versions. (Though it might work anyway while the preview is nearing release and not as many things are changing.)

All this will stabilize after the final SDK is released. Then I expect NodeRT will publish pre-built packages for each WinRT namespace (as they have done for past SDK versions), and hopefully all that generated code can be deleted from this repo.

**Comment by farfromrefug** [comment]
@jasongin i didn't know those files where auto generated!
ok so for now this PR makes no sense then.
Will close it then.

Thanks for the explanation

**Comment by jasongin** [comment]
Actually, I forgot I already had to make some edits to those generated binding.gyp files. And, the gyp files don't normally change when regenerating -- it is the C++ code that is likely to change. So I will consider this PR. I'll try to do a more thorough review in a day or two.

File: uwp/windows.devices.radios/binding.gyp [hunk 1]
```diff
@@ -1,9 +1,5 @@
 {
-  'variables': {
-    'WIN_VER' : 'v10',
-    'USE_ADDITIONAL_WINMD' : 'true'
-  },
-  'includes': [ 'common.gypi' ],
+  'includes': [ ../../'common.gypi' ],
```
Comment (jasongin) [inline uwp/windows.devices.radios/binding.gyp]: Syntax error on this line: the first quote is in the wrong place.

**Comment by jasongin** [comment]
This is great, thanks!

I went ahead and fixed the syntax error, did a test build, and merged into develop.
Then I did more testing and merged all the changes in develop up to master. This one landed as https://github.com/jasongin/noble-uwp/commit/37a309398730184fae034bca547cb6c785fa4a40

I will probably check in an update to a newer SDK version soon, and this makes it easier!

**Comment by farfromrefug** [comment]
@jasongin thanks that s great. I have to find the  time to try and build again. Will report
Thanks again

**Pull request closed without merging**